src/lib/mongoBackend/mongoUpdateContextSubscription.cpp

```diff
@@ -231,6 +231,24 @@ HttpStatusCode mongoUpdateContextSubscription
     }
   }
 
+  /* Description */
+  if (requestP->descriptionProvided)
+  {
+    // Note that in the case of description "" the field is deleted
+    if (requestP->description != "")
+    {
+      newSub.append(CSUB_DESCRIPTION, requestP->description);
+    }
+  }
```

Comment:
I might be wrong, but it seems like the 'descriptionProvided' isn't changing anything at all.
Only reason to have it would be to allow an empty description, but that is not the case ...

It seems to be you could simply compare requestP->description with "" and forget about descriptionProvided ... ?